### ISSUE TITLE
API testing: Accessibility issues

### ISSUE THREAD

**Issue by connorscarolyns** [cds-snc/platform-forms-client#4715]
**The H4 in this dialog should be an H3.** 
For accessibility reasons, do not skip heading levels to achieve sizing specs, use classes instead.

<img width="589" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBM1R1Qnc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--cedab84017fd952fe83dfaf24a43c29b9d513d1d/image.png" alt="image.png" />

<img width="577" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBOW51Qnc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--57b2118e1f014ef1c785352ad1e70a78e1cae302/image.png" alt="image.png" />

**Broken Link**
<img width="728" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBOXZ1Qnc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--f186e4350a3e29acf2c146ed5158cec6e6056d75/image.png" alt="image.png" />
